grafana-plugin/src/containers/IRMBanner/IRMBanner.tsx

```diff
@@ -15,9 +15,7 @@ const IRMBanner: React.FC = observer(() => {
   } = store;
 
   useEffect(() => {
-    if (store.isOpenSource()) {
-      alertGroupStore.fetchIRMPlan();
-    }
+    alertGroupStore.fetchIRMPlan();
```

Comment:
@teodosii I believe this should be `if (!store.isOpenSource()) {`, correct? I've moved that logic to the `alertGroupStore.fetchIRMPlan` method.